ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -824,6 +836,7 @@ public void alterTable(String catName, String dbName, String tblName, Table newT
       throw new HiveException("Unable to alter table. " + e.getMessage(), e);
     } catch (TException e) {
       throw new HiveException("Unable to alter table. " + e.getMessage(), e);
+    } finally {
```

Comment:
Empty finally?

Reply:
Rebased with master and only kept relevant changes.